west.yml

```diff
@@ -102,7 +102,7 @@ manifest:
       revision: 1c4fdba512b268033a4cf926bddd323866c3261a
       path: tools/net-tools
     - name: hal_nxp
-      revision: 12a6084be5710ec39d1cc7626582a3c470db1459
+      revision: pull/67/head
```

Comment:
The hal PR has now been merged, so you can update this to the new SHA.

Reply:
Updated.